test/core/symbolFactoriesTests.ts

```diff
@@ -0,0 +1,103 @@
+///<reference path="../testReference.ts" />
+
+describe("SymbolFactory", () => {
+  describe("SymbolFactory generates correct path", () => {
```

Comment:
Can we remove "SymbolFactory" from this describe?  When fleshing out the test name, this evaluates to
"SymbolFactory SymbolFactory generates correct path" meaning that the SymbolFactory word is redundant